app/models/order_import.rb

```diff
@@ -196,6 +198,7 @@ def set_error_mode
   def store_error_report
     self.error_file = StoredFile.new(
       file: StringIO.new(self.error_report),
+      file_content_type: "text/csv",
```

Comment:
Added a task called `paperclip:mime:update_from_file_extensions`. It should be able to run before or after the S3 migration.